src/selenium/sessionPage.js

```diff
@@ -0,0 +1,43 @@
+var BasePage = require('./basePage.js');
```

Comment:
@agbilotia1998 Good job! Lets add tests for jumping to speakers page, if the speaker image is clicked. Track if track is clicked & room when a venue is clicked.

Reply:
Okay @geekyd .